roles/backup/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Ensure python pip and paramiko dependencies are installed
+  apt:
+    name:
+      - python-pip
+      - python-dev
+      - libffi-dev
+    state: present
+- name: Ensure paramiko is installed
+  pip: name=paramiko state=present
+
+- name: List backup jobs
+  set_fact:
+    backup_user: "{{ admin_user }}"
+    backup_group: sudo
+
+    backup_mysql_user: "{{ mysql_root_user }}"
+    backup_mysql_pass: "{{ mysql_root_password }}"
+
+    # Define the backup jobs
+    backup_profiles:
+    # Backup uploads
+      - name: "{{ item.key }}_uploads"
+        schedule: "{{ item.value.backup.schedule if site_auto_backup else '0 3 31 2 *' }}" # On February 31 if not auto
```

Comment:
There ideally shouldn't be a default schedule here. Can this be omitted?

```
schedule: "{{ item.value.backup.schedule | default(omit) }}"
```

`schedule` will be skipped when `item.value.backup.schedule` is not defined